pkg/controller/apis/config/v1alpha1/defaults.go

```diff
@@ -239,6 +239,15 @@ func SetDefaults_ResourceQuotaControllerConfiguration(obj *kubectrlmgrconfigv1al
 	}
 }
 
+func SetDefaults_SAControllerConfiguration(obj *kubectrlmgrconfigv1alpha1.SAControllerConfiguration) {
+	if obj.ConcurrentSATokenSyncs == 0 {
```

Comment:
It seems this is already being set in SetDefaults_KubeControllerManagerConfiguration above.While the defaults seem to be the same do we want to allow for competing defaults?